app/scripts/modules/core/src/pipeline/config/stages/core/IStageConfigProps.ts

```diff
@@ -4,6 +4,7 @@ import { IStage } from 'core/domain';
 export interface IStageConfigProps {
   application: Application;
   stage: IStage;
+  configuration: any;
```

Comment:
I don't think this needs to be here... just in `PreconfiguredJobStageConfig.tsx` something like `export interface IPreconfiguredJobStageConfigProps extends IStageConfigProps { configuration: waitForCompletion: ?, parameters: ? }`?

Reply:
Since it's passed in from [stage.module.js](https://github.com/spinnaker/deck/pull/6152/files#diff-f0f41e28f10844cbe8077af8dfc669edR203) and a few other stages have it, it belongs further up?

I think it should be optional though, so I'll change that.

Reply:
ah, didn't realize a few other stages have it since it was just added here. carry on.